### PR TITLE
Create dude.js

### PR DESCRIPTION
**Pull request by dude22072**
New command: dude

### CODE REVIEW

**Comment by arya-s** [comment]
what's the usecase for this?

**Comment by dude22072** [comment]
It's been a long-running meme that several members of saw's server will say my name as dude{random numbers}, so I've made it easier on them.

File: src/commands/dude.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import config from '../../config';
+import { getRandom } from '../util';
+
+class Command {
+  constructor() {
+    this.aliases = ['dude'];
+  }
```
Comment (arya-s) [inline src/commands/dude.js]: We are not running out of new lines :)

File: src/commands/dude.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  run(payload) {
+    var response="dude";
```
Comment (arya-s) [inline src/commands/dude.js]: Strings should use single quotes.
Please add whitespace between `=`.

File: src/commands/dude.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    var length=getRandom(5, 100);
```
Comment (arya-s) [inline src/commands/dude.js]: Please add whitespace between `=`.

File: src/commands/dude.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    for(var i=0;i<length;i++) {
```
Comment (arya-s) [inline src/commands/dude.js]: Please add whitespace.

File: src/commands/dude.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+        response+=getRandom(0,9);
```
Comment (arya-s) [inline src/commands/dude.js]: Please add whitespace.

File: src/commands/dude.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    }
+    response += "?";
```
Comment (arya-s) [inline src/commands/dude.js]: Strings should use singlequotes.

**Review by arya-s — CHANGES_REQUESTED** [review]

**Pull request merged**